modules/hydra-master.nix

```diff
@@ -124,6 +124,20 @@ in {
     package = pkgs.postgresql96;
     dataDir = "/var/db/postgresql-${config.services.postgresql.package.psqlSchema}";
   };
+  services.influxdb = {
+    enable = true;
+  };
+  services.grafana = {
+    enable = true;
+    users.allowSignUp = true;
+    domain = "hydra.iohk.io";
+    rootUrl = "%(protocol)s://%(domain)s/grafana/";
+    extraOptions = {
+      AUTH_GOOGLE_ENABLED = "true";
+      AUTH_GOOGLE_CLIENT_ID = "778964826061-5v0m922g1qcbc1mdtpaf8ffevlso2v7p.apps.googleusercontent.com";
```

Comment:
Are these credentials attached to a shared google account set up by IT?

Reply:
it's currently on my google account. Figure we can test and make sure it works before involving Jeremy to create an oauth client at the org level and then just PR the change of the client id and update the secret.